CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
 # CHANGELOG
 
 ## vNext
-- ...
+...
+
+## 0.17.1
```

Comment:
You should increase minor and not patch.

Reply:
This is a minor version on the 2nd place

Reply:
Ok, you've increased minor and patch. Please, correct the version to `0.17.0`

Reply:
fixed